pkg/scheduler/eventhandlers_test.go

```diff
@@ -145,6 +145,38 @@ func TestNodeTaintsChanged(t *testing.T) {
 	}
 }
 
+func TestNodeAnnotationsChanged(t *testing.T) {
```

Comment:
Please implement the test as a case for TestNodeSchedulingPropertiesChange

Reply:
> Please implement the test as a case for TestNodeSchedulingPropertiesChange

Done.

Reply:
I think @alculquicondor meant to implement the test cases in TestNodeSchedulingPropertiesChange and remove this TestNodeAnnotationsChanged.

Reply:
Thanks @sanposhiho , I didn't notice that advice had been given before at  https://github.com/kubernetes/kubernetes/pull/118389/files#r1402207620